src/cmd/create.go

```diff
@@ -69,19 +69,19 @@ func init() {
 		"container",
 		"c",
 		"",
-		"Assign a different name to the toolbox container.")
+		"Assign a different name to the toolbox")
 
 	flags.StringVarP(&createFlags.image,
 		"image",
 		"i",
 		"",
-		"Change the name of the base image used to create the toolbox container.")
+		"Change the name of the base image used to create the toolbox container")
 
 	flags.StringVarP(&createFlags.release,
 		"release",
 		"r",
 		"",
-		"Create a toolbox container for a different operating system release than the host.")
+		"Use specific release of the host operating system for toolbox container")
```

Comment:
Nitpick: If we are to change the wording for the `release` flag, then it would be nice if the description for the different `release` flags were somehow similar. eg., this change doesn't match with ...